Controllers/DatasetController.py

```diff
@@ -111,25 +111,22 @@ def deleteDataset(dataset_id):
 @dataset.route("/search/<searchQuery>", methods=['GET'])
 def search(searchQuery):
     datasets = []
-    expectedBrowseReferrer = "http://localhost:8080/browse"
-    expectedManageReferrer = "http://localhost:8080/browse/manage"
-    referrerURL = request.headers["Referer"]
-    
+    browseURL = "http://localhost:8080/browse"
```

Comment:
This is not going to be localhost in prod. Need to remove the domain and just use the "/browse" or "/manage" part